src/mito2/src/engine/append_mode_test.rs

```diff
@@ -113,7 +113,7 @@ async fn test_append_mode_compaction() {
         .await
         .unwrap();
 
-    // Flush 2 SSTs for compaction.
+    // Flush 3 SSTs for compaction.
```

Comment:
was the comment incorrect or we change this test?

Reply:
The comment is outdated.